github/pulls_reviews.go

```diff
@@ -65,9 +65,17 @@ func (r PullRequestReviewDismissalRequest) String() string {
 // Read more about it here - https://github.com/google/go-github/issues/540
 //
 // GitHub API docs: https://developer.github.com/v3/pulls/reviews/#list-reviews-on-a-pull-request
-func (s *PullRequestsService) ListReviews(ctx context.Context, owner, repo string, number int) ([]*PullRequestReview, *Response, error) {
+func (s *PullRequestsService) ListReviews(ctx context.Context, owner, repo string, number int, opt *ListOptions) ([]*PullRequestReview, *Response, error) {
 	u := fmt.Sprintf("repos/%v/%v/pulls/%d/reviews", owner, repo, number)
 
+	if opt != nil {
```

Comment:
No need to check that opt != nil, because `addOptions` handles that case.

See https://github.com/google/go-github/blob/c03cf196876fc0b7cf8db2516848536778ea4e29/github/github.go#L182-L184.

Please use the same style as all other places that call `addOptions`. You can use [`RepositoriesService.ListCommits`](https://github.com/google/go-github/blob/c03cf196876fc0b7cf8db2516848536778ea4e29/github/repos_commits.go#L114-L117) as reference:

```Go
u := fmt.Sprintf("repos/%v/%v/pulls/%d/reviews", owner, repo, number)
u, err := addOptions(u, opt)
if err != nil {
	return nil, nil, err
}
```